code/modules/admin/verbs/infiltratorteam_syndicate.dm

```diff
@@ -21,7 +21,7 @@ var/global/sent_syndicate_infiltration_team = 0
 	var/pick_manually = 0
 	if(alert("Pick the team members manually? If you select yes, you pick from ghosts. If you select no, ghosts get offered the chance to join.",,"Yes","No")=="Yes")
 		pick_manually = 1
-	var/list/teamsizeoptions = list(2,3,4,5,6)
+	var/list/teamsizeoptions = list(1,2,3,4,5,6)
```

Comment:
When I originally made SITs, I specifically did not have an option for sending just one, since, well, their name includes 'team' in it. If you're going to add this, be sure to test it carefully.
Also, spacing on the numbers.

Reply:
Haven't seen any issues spawning a single operator team, although I lack the means to test with anyone that is an "actual ghost" and not just myself as an observing admin. Ultimately since it's an admin tool I feel like admins should be able to send a single operator (for, say, an assassination mission) if they feel it appropriate and it's not buggy, which as far as I can tell it isn't

Reply:
In almost any case you'd want to send a single infiltrator, there's another, better option. If you want someone killed or protected, send a 'hunter' (bless/smite option). That way they get a pinpointer to the target, automatically spawn in arrivals, etc. If you want to add a traitor to the round, there's one-click-antag. There's also the ability to give custom objectives to existing antags.

SIT is really optimized for a single use-case: when you want to send a co-ordinated team of non-murderbone syndis with a special objective that isn't one of the normal types.

Yeah, you CAN edit it so its possible to send a syndi infiltrator... but I specifically left out that option to encourage admins to use the other options I felt were better for that use case.